### PR TITLE
fix broken README.md

### LINKED ISSUE #277: README is broken on NPM
## What:

NPM states that `atomic-layout` package doesn't have a README.

## Why:

Due to the monorepo structure.

## Environment:

- atomic-layout version: `0.12.3`

## Steps to reproduce:

1. Go to https://www.npmjs.com/package/atomic-layout
2. See the following message instead of README:

```
"Unable to find a readme for atomic-layout@0.12.3"
```

## Expected behavior:

`atomic-layout` package on NPM displays the current README from the repository.

## Screenshots:

Screen Shot 2019-12-30 at 12 57 48

### PR DESCRIPTION
**Pull request by ruhaise**
## Changes

- added symlink for README.md from packages to the root,

## GitHub

- Closes #277 

## Release version

- [ ] patch (internal improvements)
- [ ] minor (backward-compatible changes)
- [ ] major (breaking, backward-incompatible changes)

## Contributor's checklist

- [x] My branch is up-to-date with the latest `master`
- [x] I ran `yarn verify` and verified the build and tests passing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -152,7 +152,7 @@ Atomic Layout's browser support is made by the browser support of underlying tec
 
 Thank you for deciding to contribute! Your involvement makes a significant impact on the library and its future.
 
-**Please read the [Contribution guidelines](./.github/CONTRIBUTING.md)** to get familiar with the contributing process. The issues labeled [`help wanted`](https://github.com/kettanaito/atomic-layout/labels/help%20wanted) or [`good first issue`](https://github.com/kettanaito/atomic-layout/labels/good%20first%20issue) are a good place to start cooperating on Atomic Layout. Feature suggestions or bug reports, discussion, and pull requests are always welcome!
+**Please read the [Contribution guidelines](https://github.com/kettanaito/atomic-layout/blob/277-broken-README.md/.github/CONTRIBUTING.md)** to get familiar with the contributing process. The issues labeled [`help wanted`](https://github.com/kettanaito/atomic-layout/labels/help%20wanted) or [`good first issue`](https://github.com/kettanaito/atomic-layout/labels/good%20first%20issue) are a good place to start cooperating on Atomic Layout. Feature suggestions or bug reports, discussion, and pull requests are always welcome!
```
Comment (kettanaito) [inline README.md]: It seems you have included the absolute link relative to the feature branch. Please, could you include it relative to master branch (just as you did with other assets above)?

File: packages/atomic-layout/.github/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+../../../README.md
```
Comment (kettanaito) [inline packages/atomic-layout/.github/README.md]: What does this do? Is it some GitHub feature?

Comment (ruhaise) [reply packages/atomic-layout/.github/README.md]: @kettanaito  yes, this is how we do it to show it github.

File: README.md (hunk 1 above)
Comment (ruhaise) [reply README.md]: @kettanaito  i did that already

**Comment by ruhaise** [comment]
@kettanaito  can you please check it again, if this looks ok?

**Comment by kettanaito** [comment]
@ruhaise, rebased this one against the latest master and checked: looks great. Will wait for the CI to finish and will merge. Once more thanks for these changes!

**Pull request merged**